Tests/SwiftGRPCNIOTests/ServerThrowingTests.swift

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Dispatch
+import Foundation
+import NIO
+import NIOHTTP1
+import NIOHTTP2
+@testable import SwiftGRPCNIO
+import XCTest
+
+private let expectedError = GRPCStatus(code: .internalError, message: "expected error")
```

Comment:
nit: Maybe prefix as `kExpectedError` since this is a global constant?

Reply:
Given that this is a private fixture used in just one file, I prefer to not use the `k` prefix in this case.